mobilecoind/clients/python/blockchain_explorer/requirements.txt

```diff
@@ -0,0 +1,4 @@
+Flask==1.1.2
+grpcio==1.32.0
```

Comment:
I don't think `grpcio` or `grpcio-tools` are needed

Reply:
It wouldn't run for me until I installed those